lib/network_util.py

```diff
@@ -6,23 +6,154 @@
 import networkx as nx 
 
 
-class Graph(object):
+class Graph():
+
+    node_dict_factory = dict
```

Comment:
Is there any reason for not keeping them inside the constructor?

Reply:
https://networkx.github.io/documentation/stable/_modules/networkx/classes/graph.html

The original networkx source code does like this, we can also have them inside constructor